tools/topology/topology2/include/pipelines/cavs/dai-copier-be.conf

```diff
@@ -72,6 +72,7 @@ Class.Pipeline."dai-copier-be" {
 		copier."2" {
 			type	buffer
 			copier_type	"module"
+			node_type	$BUFFER_COPIER
```

Comment:
can't we have this attribute set as immutable in the base class? Or have a different copier-module class?

Reply:
i wanted to take the sort-cut but yes makes better sense to have a new class. fixed now